tests/utils.go

```diff
@@ -677,8 +677,8 @@ func NewRandomMigration(vmiName string, namespace string) *v1.VirtualMachineInst
 	}
 }
 
-func NewRandomVMIWithEphemeralDisk(containerImage string) *v1.VirtualMachineInstance {
-	vmi := NewRandomVMI()
+func NewRandomVMIWithEphemeralDisk(containerImage string, opts ...libvmi.Option) *v1.VirtualMachineInstance {
```

Comment:
Same, it would be better to replace it with `libvmi` instead. Or replace it at the callers when you need these options.